mw-webapp/src/dataAccessLogic/renderCellValue/helpers/updateJobDone.ts

```diff
@@ -0,0 +1,14 @@
+import {JobDoneDAL} from "src/dataAccessLogic/JobDoneDAL";
+import {JobDone} from "src/model/businessModel/JobDone";
+
+/**
+ * Update JobDone
+ */
+export const updateJobDone = async (text: string, uuid: string) => {
```

Comment:
usually its better to avoid synonims, text here using only like description, so why cant we use just `description` intead of `text`

check evcerywhere

Reply:
Changed